website/src/middlewares/api-v3/ensureDevelpmentMode.js

```diff
@@ -0,0 +1,12 @@
+import nconf from 'nconf';
+import {
+  NotFound,
+} from '../../libs/api-v3/errors';
+
+module.exports = function ensureDevelpmentMode (req, res, next) {
+  if (nconf.get('IS_PROD')) {
```

Comment:
let's define IS_PROD as a const at the beginning of the file instead of calling nconf.get at every request